crates/edr_provider/src/requests/eth/call.rs

```diff
@@ -70,7 +61,10 @@ fn resolve_call_request(
         })
     } else {
         // Question: Is there a reason we don't support EIP-2930 calls?
```

Comment:
@fvictorio & @alcuadrado?

Reply:
I guess we should, even if they are uncommon? My understanding is that an EIP-2930 transaction request should be used if the call object has both `gasPrice` and `accessList` fields.

But as always, if Hardhat is not currently doing this, then I think it's fine to open an issue for it and keep it as it is. An EIP-2930 seems like a very edge case to me.

Reply:
https://github.com/NomicFoundation/edr/issues/255